### PR TITLE
Brain

### PR DESCRIPTION
**Pull request by Antimatter543**
Big brain time!!!!

### CODE REVIEW

**Comment by Antimatter543** [comment]
brain big ,conditionals do things.
mmm use strat yes

File: python3/totoro_agent/brain/brain.py [hunk 1]
```diff
@@ -16,17 +16,50 @@ def __init__(self):
...
+            print('bomb loc', bomb_loc, 'blast tiles:', potential_blast_tiles, 'player', game_state['player_pos'])
+
+            if tuple(game_state['player_pos']) in potential_blast_tiles:
+                print('HOLY RUN FOR YOUR LIFE YOU ARE GONNA GET RAILED')
+                return 'retreat'
 
-        # TODO Make decisions in terms on what strategy to do next
+        ### Basic Decision Making
+        # Pickup if ammo, stalk if none on map.
+        if len(game_state['ammo_list']) != 0: # "Is there ammo on the map?"
```
Comment (chrisrabe) [inline python3/totoro_agent/brain/brain.py:61]: check for powerup too

**Review by chrisrabe — CHANGES_REQUESTED** [review]
need to check for pickup

**Review by chrisrabe — APPROVED** [review]
looks good

**Pull request merged**